core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -60,23 +62,26 @@
 import org.elasticsearch.test.VersionUtils;
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.threadpool.ThreadPoolModule;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
 import org.junit.*;
 
 import java.io.IOException;
+import java.util.Arrays;
 
 import static org.hamcrest.Matchers.*;
 
 @Ignore
 public abstract class BaseQueryTestCase<QB extends AbstractQueryBuilder<QB>> extends ElasticsearchTestCase {
 
-    protected static final String OBJECT_FIELD_NAME = "mapped_object";
-    protected static final String DATE_FIELD_NAME = "mapped_date";
-    protected static final String INT_FIELD_NAME = "mapped_int";
     protected static final String STRING_FIELD_NAME = "mapped_string";
+    protected static final String INT_FIELD_NAME = "mapped_int";
     protected static final String DOUBLE_FIELD_NAME = "mapped_double";
     protected static final String BOOLEAN_FIELD_NAME = "mapped_boolean";
-    protected static final String[] mappedFieldNames = new String[] { DATE_FIELD_NAME, INT_FIELD_NAME, STRING_FIELD_NAME,
-            DOUBLE_FIELD_NAME, BOOLEAN_FIELD_NAME, OBJECT_FIELD_NAME };
+    protected static final String DATE_FIELD_NAME = "mapped_date";
+    protected static final String OBJECT_FIELD_NAME = "mapped_object";
+    protected static final String[] mappedFieldNames = new String[] { STRING_FIELD_NAME, INT_FIELD_NAME,
+            DOUBLE_FIELD_NAME, BOOLEAN_FIELD_NAME, DATE_FIELD_NAME, OBJECT_FIELD_NAME };
```

Comment:
just the ordering the change, which seems more logical, first String, then Int, then Double, Boolean, Date and finally Object.